config/routes/plaatjes.yaml

```diff
@@ -0,0 +1,11 @@
+upload:
+  path: /plaatjes/upload
+  controller: CsrDelft\controller\PlaatjesController::upload
```

Comment:
Plaatjes en Plaetjes is idd wel een beetje verwarrend.

Reply:
Misschien /forum/plaatjes?